src/views/Home/components/Question/questionData.js

```diff
@@ -1,62 +1,51 @@
 import wellness from '../../../../services/wellness.js';
-let user, qOne;
+import user from '../../../../services/user';
+
+let currentUser, qOne;
 
 /*
  * Fetches the user's data
- *
- * @return {JSON} The JSON data of the current user
  */
 async function getUserData() {
-  // Gets the token from local storage to prove authentication for fetch
-  let token = JSON.parse(localStorage.getItem('token'));
-
-  // Creates the header for the request to get the user's info
-  let headers = new Headers({
-    Authorization: `Bearer ${token}`,
-    'Content-Type': 'application/json',
+  currentUser = await user.getProfileInfo().catch(error => {
+    console.log(error.message);
   });
-
-  return await fetch(
-    new Request('https://360api.gordon.edu/api/profiles', { method: 'GET', headers }),
-  )
-    .then((result) => {
-      return result.json();
-    })
-    .then((data) => {
-      user = data;
-    })
-    .catch((error) => {
-      console.log(error.message);
-    });
 }
 
 export async function getQuestions() {
-  
   await getUserData();
   let backendQuestions = await wellness.getQuestion();
 
-  let phoneNumber = `(${user.MobilePhone.substring(0, 3)}) ${user.MobilePhone.substring(
+  let phoneNumber = `(${currentUser.MobilePhone.substring(
+    0,
```

Comment:
this code has changed and is yet to be pushed so it will eventually get replaced.

Reply:
Sounds good!